server/datastore/mysql/hosts.go

```diff
@@ -628,12 +628,13 @@ func (ds *Datastore) EnrollHost(ctx context.Context, osqueryHostID, nodeKey stri
 					detail_updated_at,
 					label_updated_at,
 					policy_updated_at,
+					last_enrolled_at,
```

Comment:
it's weird that we weren't setting this value here, right?

Reply:
Indeed, makes sense to set it when creating the host at `EnrollHost`, right?

Reply:
This defaults to CURRENT_TIMESTAMP when not specified though. Are you sure this is needed?

Reply:
Ah good catch, I'll revert then.